### ISSUE TITLE
Regression: Notification sounds do not play [Windows]

### ISSUE THREAD

**Issue by ZDBioHazard** [mattermost/desktop#1606]
I confirm (by marking "x" in the [ ] below: [x]):

- [x] I have confirmed this is an issue on the newest beta build [available here](https://github.com/mattermost/desktop/tags)
- [x] This issue doesn't reproduce on web browsers (such as in Chrome). If it does, [issue reports go to the Mattermost Server repository](https://github.com/mattermost/platform/issues).
- [x] I have read [contributing guidelines](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md).

---

**Summary**
When a user receives a notification, the configured notification sound does not play.

**Environment**
- Operating System: Windows 10 LTSC
- Mattermost Desktop App version: 4.7.0-rc4 (EXE and MSI both tested)
- Mattermost Server version: 5.34.2 (via Docker)

**Steps to reproduce**
- Enable notification sounds.
- Receive a message that generates a notification.

**Expected behavior**
Desktop notification pops up and notification sound plays.

**Observed behavior**
Desktop notification pops up, but notification sound does not play.

The following errors appear in the application wrapper console each time a notification is generated:
```
* Refused to load media from 'data:audio/mpeg;base64,[...]' because it violates the following Content Security Policy directive: "default-src 'self'". Note that 'media-src' was not explicitly set, so 'default-src' is used as a fallback.
* Uncaught (in promise) DOMException: Failed to load because no supported source was found.
```

**Possible fixes**
The Content-Security-Policy appears to be set as a `<META>` tag in `/src/renderer/index.html:5`. It was changed between the last tried working version (4.6.2) and the affected version, so maybe that has something to do with it.

[The Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/media-src) caution against allowing `data:` URIs as a `media-src` though, so watch out:
> This is insecure; an attacker can also inject arbitrary data: URIs. Use this sparingly and definitely not for scripts.

**Comment by Willyfrog** [comment]
Thanks for reporting!! There is a ticket for that already: https://mattermost.atlassian.net/browse/MM-35662

**Comment by Willyfrog** [comment]
this should be fixed in https://github.com/mattermost/desktop/pull/1615

**Comment by ZDBioHazard** [comment]
I can confirm that this issue is fixed in `4.7.0-rc6`
